paddle/fluid/framework/new_executor/interpreter/interpreter_util.h

```diff
@@ -75,9 +76,8 @@ void BuildOpFuncList(const platform::Place& place,
                      const std::set<std::string>& skip_gc_vars,
                      std::vector<OpFuncNode>* vec_func_list,
                      VariableScope* scope,
-                     bool use_local_scope = true,
-                     bool used_for_jit = false,
-                     bool used_for_control_flow_op = false);
+                     ExecutionConfig execution_config,
```

Comment:
ExecutinoConfig contains a member of skip_gc_vars set that value copying may have high overhead. Therefore, better to use const reference for parameter ExecutionConfig here.

Reply:
Done.